ci-operator/config/openshift/logging-view-plugin/openshift-logging-view-plugin-release-5.6.yaml

```diff
@@ -0,0 +1,51 @@
+base_images:
+  nginx-120:
+    name: ubi8-nginx
```

Comment:
As we are not using nginx anymore, maybe it will make sense to have a nodejs base image like: https://catalog.redhat.com/software/containers/ubi8/nodejs-16/615aee9fc739c0a4123a87e1, this will allow to simplify the `Docker.ci`